index.html

```diff
@@ -924,8 +924,11 @@ <h2>Terms of Use</h2>
         <dl>
           <dt><var>termsOfUse</var></dt>
           <dd>The value of this property MUST be one or more terms of use
-            descriptions that provide enough information to a <a>verifier</a>
+            that provide enough information to a <a>verifier</a>
             to determine how they may utilize the given information.
+            Each terms of use is uniquely identified by a type and id. The precise
```

Comment:
While `id` is preferred, it is not required. Nodes in the graph that are not assigned an `id` have one auto-generated for them during canonicalization or JSON-LD processing. We should make sure that people know that `id` is optional via a SHOULD. Also, keep in mind that the ODRL community will probably have more than a few thoughts on this as well.

Reply:
@msporny . When you say id is not required, this may be the case for pure JSON-LD. But in the VC spec we can mandate the use of the id if we want to. In other words, the VC spec can place constraints on JSON-LD when it is deemed to be appropriate. For example, a VC must itself always have a type and id property.

Reply:
> But in the VC spec we can mandate the use of the id if we want to.

Yes, I'm stating that I don't think id is necessary here because you're describing potentially transient information and thus referring to it by `id` is something we're forcing developers to do when they will end up just creating a random throw-away identifier (like a UUID).

Reply:
1. can ToU "attach" (using that term loosely) to any of our artifacts: Verifiable Credential or Presentation?
1. can ToU be added by either Issuer or Holder?
1. is the "owner/setter" of the ToU clear?